MediaFeeder/MediaFeeder/PlaybackManager/PlaybackSession.cs

```diff
@@ -15,10 +16,14 @@ public sealed class PlaybackSession : IDisposable
     private int? _volume;
     private float? _rate;
     private float? _loaded;
+    public string? Title { get; set; }
     public event Action? UpdateEvent;
     public event Action? PlayPauseEvent;
     public event Action? WatchEvent;
     public event Action? SkipEvent;
+    public List<Folder> AllFolders { get; set; } = [];
```

Comment:
Is this used in the session at all, or only on the session info page? If not, can it be moved?